server/catgenome/src/main/java/com/epam/catgenome/manager/reference/ReferenceManager.java

```diff
@@ -570,6 +590,20 @@ private long registerReference(Long referenceId, Reference reference, boolean cr
         return lengthOfGenome;
     }
 
+    @SneakyThrows
+    private String genbankToFasta(final Reference reference) {
+        String genbankFilePath = reference.getPath();
+        Assert.notNull(genbankFilePath, getMessage(MessageCode.RESOURCE_NOT_FOUND));
+        File genbankFile = new File(genbankFilePath);
```

Comment:
we also need to support possibility to work with remote file, see `FastaSequenceFile`
for this point we can use `SeekableStreamFactory.getInstance().getStreamFor(path)` to be able to get InputStream from some resource type, and use `GenbankReaderHelper.readGenbankDNASequence` that gets `InputStream` as input instead of File